src/herbpy/action/rogue.py

```diff
@@ -0,0 +1,354 @@
+def HaltHand(robot, manip=None):
+    """
+    @param robot The robot being used for the stopping gesture
+    @param manip The manipulator being used for the stopping gesture
+    """
+
+    if manip == None:
+        manip = robot.GetActiveManipulator()
+
+    if manip.GetName() == 'right':
+        pose = numpy.array([5.03348748, -1.57569674,  1.68788069,
+                            2.06769058, -1.66834313,
+                            1.53679821,  0.21175342])
+        
+        manip.PlanToConfiguration(pose, execute=True)
+        robot.right_hand.MoveHand(f1=0, f2=0, f3=0, spread=3.14)
+    elif manip.GetName() == 'left':
+        pose = numpy.array([ 1.30614268, -1.76      , -1.57063853,
+                             2.07228362,  1.23918377,
+                             1.46215605, -0.12918424])
+
+        manip.PlanToConfiguration(pose, execute=True)
+        robot.left_hand.MoveHand(f1=0, f2=0, f3=0, spread=3.14)
+    else: 
+        raise prpy.exceptions.PrPyException('Stop is only defined \
+                                                for the left and right arm.')
+
+@ActionMethod
+def MiddleFinger(robot, manip=None):
+    """
+    @param robot The robot being used to give the middle finger
+    @param manip The manipulator being used to give the middle finger.
+                 Must be either the right or left arm.
+    """
+    if manip == None:
+        manip = robot.GetActiveManipulator()
+
+    if manip.GetName() == 'right':
+        right_dof = numpy.array([ 5.03348748, -1.57569674,  1.68788069,  
+                                  2.06769058, -1.66834313,
+                                  1.53679821,  0.21175342])
+         
+        manip.PlanToConfiguration(right_dof, execute=True)
+        robot.right_hand.MoveHand(f1=2, f2=2, f3=0, spread=3.14)
+
+    elif manip.GetName() == 'left':
+        left_dof = numpy.array([ 1.30614268, -1.76      , -1.57063853,  
+                                 2.07228362,  1.23918377,
+                                 1.46215605, -0.12918424])
+
+        manip.PlanToConfiguration(left_dof, execute=True)
+        robot.left_hand.MoveHand(f1=2, f2=2, f3=0, spread=3.14)
+    else: 
+        raise prpy.exceptions.PrPyException('The middle finger is only defined \
+                                for the left and right arm.')
+
+
+@ActionMethod
+def Wave(robot):
+    """
+    @param robot The robot waving with their right arm
+    """
+
+    from prpy.rave import load_trajectory
+    from prpy.util import FindCatkinResource
+    from os.path import join
+    env = robot.GetEnv()
+
+    wave_path = FindCatkinResource('herbpy', 'config/waveTrajs/')
+    try:
+        traj0 = load_trajectory(env, join(wave_path, 'wave0.xml'))
+        traj1 = load_trajectory(env, join(wave_path, 'wave1.xml'))
+        traj2 = load_trajectory(env, join(wave_path, 'wave2.xml'))
+        traj3 = load_trajectory(env, join(wave_path, 'wave3.xml'))
+    except IOError as e:
+        raise IOError(str(e))
```

Comment:
This does nothing. If you don't want to change the error message, then just remove the `try`-`except` block entirely.